Cargo.toml

```diff
@@ -39,6 +39,6 @@ arrow-select = { version = ">=53, <55" }
 arrow-schema = { version = ">=53, <55" }
 parquet = { version = ">=53, <55", features = ["object_store"] }
 object_store = { version = ">=0.11, <0.12" }
-hdfs-native-object-store = "0.12.0"
-hdfs-native = "0.10.0"
```

Comment:
Hi @rzepinskip, thanks for your contribution!

I may be missing something, but doesn't the compatibility matrix state that hdfs-native=0.11 is only compatible with hdfs-native-obiject-store=0.13.x?

Reply:
Right, I had 0.13.0 there before but I found out it is not released yet and changed to 0.12.1. I raised a [question](https://github.com/datafusion-contrib/hdfs-native-object-store/issues/14) in hdfs-native-object-store repo to clarify confusion.